packages/hooks/src/useTotalAccountBalance.ts

```diff
@@ -0,0 +1,30 @@
+import { useStore } from '@deriv/stores';
+
+/**
+ * we can use this hook to get the total balance of the given accounts list.
+ * it loops through the accounts list and adds the balance of each account
+ * to the total balance, it also converts the balance to the currency of the
+ * first account in the list
+ */
+
+const useTotalAccountBalance = (accounts: { balance?: number; currency?: string }[]) => {
+    const { exchange_rates, client } = useStore();
+    const { current_fiat_currency, is_crypto, currency, default_currency } = client;
+    const currency_if_is_crypto = current_fiat_currency ?? default_currency;
```

Comment:
```suggestion
    const currency_if_is_crypto = current_fiat_currency || default_currency;
```
🤔

Reply:
both are the same in this case.

Reply:
Not the same man 👀 

<img width="103" alt="image" src="https://user-images.githubusercontent.com/72082844/231659208-7a639a74-94de-4787-8a22-9540ed26a6a6.png">

Reply:
Oh, I didn't check for empty string :smile:  updated. good catch